packages/amplify_authenticator/lib/src/widgets/button.dart

```diff
@@ -244,111 +273,12 @@ class BackToSignInButton extends StatelessAuthenticatorComponent {
   }
 }
 
-class GoToSignUpButton extends StatelessAuthenticatorComponent {
```

Comment:
These next two buttons are not used. I think they could be useful for the Custom UI using authenticatorBuilder, but they haven't been tested (to my knowledge) in a while. We can always add them back if we want.